content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title:       "ReactOS in 2020"
+author:      "Extravert34"
+date:        2020-12-28
+---
+
+Despite all the turbulence, we've got quite a rich year for ReactOS.
+Many bugs and instabilities were resolved, many more have been introduced. This year we hired two kernel developers full-time, this happens for the first time on the project history.
+
+The post highlights some of the changes which may may be interesting to the community.
+
+## Shell changes
+
+Shell doesn't see much attention recently, due to most of the work concentrated in the kernel, but there are still some useful fixes and feature implementations.
+
+* Katayama Hirofumi MZ made a great job implementing the Filesystem change notification feature (aka `SHChangeNotify`) in the shell. This is required for all applications monitoring the filesystem changes
+* Visual and usability improvements: "Size on disk" label in a file Properties dialog, key accelerators in Open/Save dialog and many more
+
+## RAPPS (a GSoC project)
+
+This is one of our three GSoC projects of the year. He Yang continues the work from previous GSoC years, by improving our RAPPS application manager. This includes UI & usability enhancements, showing app screenshots, command line scripting support.
+
+*picture*
```

Comment:
You can use the picture from the gsoc post, and even link to it:
img/blogs/gsoc2020-rapps-final-report.png